staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -123,6 +123,7 @@ type DebugOptions struct {
 	TargetContainer string
 	TTY             bool
 	Profile         string
+	Applier         ProfileApplier
```

Comment:
could you help me catch up on why this is now exported?

Reply:
Arda suggested this during review of this MR:

![image](https://user-images.githubusercontent.com/18542826/214217071-ee3f79df-966a-4bf0-ac9a-04e0cc9c0ecb.png)

Reply:
This is basically for the other CLI's like `oc` to use `kubectl debug`. `oc` has it's own custom profiles and it will set `Applier` with custom profiles.